config/routes.rb

```diff
@@ -19,19 +19,13 @@
     get  'import_progress'
   end
 
-  resources :tag_migrations, only: %i(index show create) do
+  resources :tag_migrations, only: [:index, :new, :create, :show, :destroy] do
```

Comment:
Tempted because it's shorter, but I always find it easier to see whats available when the `only` form is used.